recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,146 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._exceptions import RecipeScrapersExceptions
+from ._utils import get_minutes, normalize_string
+
+
+class UnsupportedLocale(RecipeScrapersExceptions):
+    """No support for selected locale of this website"""
+
+    def __init__(self, lang):
+        self.lang = lang
+        message = f'Selected locale "{self.lang}" is not supported.'
```

Comment:
Informationally for the user, I think it's important that we clarify that the limited locale support is for (at least) this website.

In fact, now that I think about it; it may be better to specify the URL of the webpage -- because the language of a webpage (or the recipe included in it) doesn't always match the top-level domain.

```suggestion
    """No support for detected recipe locale on this webpage"""

    def __init__(self, lang, url):
        self.lang = lang
        self.url = url
        message = f'Detected recipe locale "{self.lang}" is not supported for url={url}'
```